### PR TITLE
Adding UniFi WLAN Password sensor

### PR DESCRIPTION
**Pull request by brunohenriquy**
## Proposed change

I have added a new sensor, WLAN Password, to the UniFi integration. This sensor (disabled by default) allow users to retrieve the password of a WLAN. This addition enhance the functionality of the UniFi integration by providing users with more control over their network configurations.
- Accessibility
It's quite helpful for accessibility purposes. By integrating it into voice assistants like Alexa, users with visual impairments can simply ask for the WLAN password, and the assistant can speak it aloud. This makes accessing the network much easier for everyone, including blind individuals. Additionally, not everyone can scan the QR code at all times.
- Alternative method for accessing the password
Sometimes, you may need the password to enter it manually on a laptop or other devices.
- Guest WLAN security
Imagine an automation that resets the guest password (upcoming feature) every day and displays the new one along with the QR code on a TV in a small business. This setup would prevent people nearby from staying connected indefinitely and ensure that only daily clients use the WiFi.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/114419

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/unifi.markdown [hunk 1]
```diff
@@ -152,6 +152,13 @@ Get entities reporting the general temperature of a UniFi Network device.
 
 Get entities reporting the current state of a UniFi Network device.
 
+### WLAN Password sensor
+
+Provides the current password for a specific WLAN configured in a UniFi Network device. This sensor allows users to retrieve the WLAN password for various purposes, such as sharing with guests or configuring devices manually.
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
Provides the current password for a specific WLAN. This sensor allows users to retrieve the WLAN password for various purposes, such as sharing with guests or configuring devices manually.
```
WLANs doesn't have any direct relation to a device

File: source/_integrations/unifi.markdown [hunk 2]
```diff
@@ -152,6 +152,13 @@ Get entities reporting the general temperature of a UniFi Network device.
 
 Get entities reporting the current state of a UniFi Network device.
 
+### WLAN Password sensor
+
+Provides the current password for a specific WLAN. This sensor allows users to retrieve the WLAN password for various purposes, such as sharing with guests or configuring devices manually.
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]: I would like to clarify that this is more suggested as a fallback when the QR code is not applicable

Comment (brunohenriquy) [reply source/_integrations/unifi.markdown]: Check my adjusted version.

File: source/_integrations/unifi.markdown [hunk 3]
```diff
@@ -152,6 +152,13 @@ Get entities reporting the general temperature of a UniFi Network device.
 
 Get entities reporting the current state of a UniFi Network device.
 
+### WLAN Password sensor
+
+Provides the current password for a specific WLAN. This sensor serves as a fallback option for retrieving the WLAN password when the QR code is not applicable. It allows users to access the password for various purposes, such as sharing with guests or configuring devices manually.
```
Comment (Kane610) [inline source/_integrations/unifi.markdown]:
```suggestion
Provides the current password for a specific WLAN. It allows users to access the password for various purposes, such as sharing with guests or configuring devices manually. As this exposes the password in clear text it is only recommended to utilize this in scenarios where the QR Code is not sufficient. 
```

**Review by Kane610 — DISMISSED** [review]

**Review by Kane610 — APPROVED** [review]

**Pull request merged**